how-to-pitch/readme.md

```diff
@@ -0,0 +1,49 @@
+> Goal of this guide: Helping you to pitch
+
+# How to pitch
+
+"Trust me I got this" is one of the least reassuring sentences in the world. So how do you convey that you can do something without clearly saying that you "got diss"?
+
+<img src="https://media.giphy.com/media/XfxbrDXAkIqFa/giphy.gif" width="100%">
```

Comment:
I like the grainy ness of it, so id leave it like this unless you think it is not publishable